odissei/np/README.md

```diff
@@ -0,0 +1,3 @@
+Nanopublication
+===============
+
```

Comment:
What is the purpose of this file? It does not really add any information, can it be deleted?

Reply:
I have deleted the README files as recommended